Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py

```diff
@@ -13,342 +16,411 @@
+		result[demisto_closing_reason] = arcanna_label
 
-    return result
+	return result
 
 
 def main() -> None:
-    """main function, parses params and runs command functions
-
-    :return:
-    :rtype:
-    """
-
-    api_key = demisto.params().get('apikey')
-
-    # get the service API url
-    base_url = urljoin(demisto.params()['url'])
-    verify_certificate = not demisto.params().get('insecure', False)
-    feature_mapping = demisto.params().get('feature_mapping')
-    proxy = demisto.params().get('proxy', False)
-
-    default_job_id = demisto.params().get('default_job_id', -1)
-    demisto.debug(f'Command being called is {demisto.command()}')
-    try:
-
-        client = Client(
-            api_key=api_key,
-            base_url=base_url,
-            verify=verify_certificate,
-            proxy=proxy,
-            default_job_id=default_job_id
-        )
-
-        if demisto.command() == 'test-module':
-            # This is the call made when pressing the integration Test button.
-            result_test = test_module(client, feature_mapping)
-            return_results(result_test)
-        elif demisto.command() == "arcanna-get-jobs":
-            result_get_jobs = get_jobs(client)
-            return_results(result_get_jobs)
-        elif demisto.command() == "arcanna-send-event":
-            result_send_event = post_event(client, demisto.args())
-            return_results(result_send_event)
-        elif demisto.command() == "arcanna-get-event-status":
-            result_get_event = get_event_status(client, demisto.args())
-            return_results(result_get_event)
-        elif demisto.command() == "arcanna-get-default-job-id":
-            result_get_default_id = get_default_job_id(client)
-            return_results(result_get_default_id)
-        elif demisto.command() == "arcanna-set-default-job-id":
-            result_set_default_id = set_default_job_id(client, demisto.args())
-            return_results(result_set_default_id)
-        elif demisto.command() == "arcanna-send-event-feedback":
-            result_send_feedback = send_event_feedback(client, feature_mapping, demisto.args())
-            return_results(result_send_feedback)
-        elif demisto.command() == "arcanna-send-bulk-events":
-            result_bulk = send_bulk_events(client, feature_mapping, demisto.args())
-            return_results(result_bulk)
-        elif demisto.command() == "arcanna-get-feedback-field":
-            result_feedback_field = get_feedback_field(demisto.params())
-            return_results(result_feedback_field)
-    # Log exceptions and return errors
-    except Exception as e:
-        demisto.error(traceback.format_exc())  # print the traceback
-        return_error(f'Failed to execute {demisto.command()} command.\nError:\n{str(e)}')
+	"""main function, parses params and runs command functions
+
+	:return:
+	:rtype:
+	"""
+
+	api_key = demisto.params().get('apikey')
```

Comment:
It's our convention now to use `credentials` for authentication.
https://xsoar.pan.dev/docs/integrations/code-conventions#credentials
We're strongly recommend to update this